### PR TITLE
Added Django Generator (with rest framework, no provisioning or front…

### PR DESCRIPTION
**Pull request by BenEllerby**
(with rest framework, no provisioning or frontend)
- Still to do
 - Add react integration for frontend.
 - Add eb deployment.
 - Add pylint

### CODE REVIEW

File: generators/app/templates/django_bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+echo 'ben'
```
Comment (Miniplop) [inline generators/app/templates/django_bootstrap.sh]: maybe this line isn't very usefull :smile_cat:

File: generators/app/index.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ class StackGenerator extends Generator {
         name    : 'backend',
         message : 'Choose your backend',
         default : 'API Platform (Symfony)',
-        choices : ['API Platform (Symfony)', 'Loopback (nodejs)', 'none']
+        choices : ['API Platform (Symfony)', 'API Platform (Django)', 'Loopback (nodejs)', 'none']
```
Comment (cRicateau) [inline generators/app/index.js]: API Platform is a framework (https://github.com/api-platform/demo) => You could name it Djando (python)

**Comment by cRicateau** [comment]
@BenEllerby thanks for contributing! Django should definitely be in this generator. Do you use docker or vagrant at theodo UK ?

**Comment by cRicateau** [comment]
I first tried it in a directory called 'django', it failed because the folder name was conflicting with the package django.

File: generators/app/index.js [hunk 3]
```diff
@@ -383,23 +387,62 @@ class StackGenerator extends Generator {
     return Promise.resolve();
   }
 
+
+  _addDjangoServer () {
+    this.spawnCommandSync(this.templatePath('django_bootstrap.sh'), [this.answers.appName]);
+    const settingsPath = `${this.answers.appName}/${this.answers.appName}`;
```
Comment (cRicateau) [inline generators/app/index.js]: could you clone it in a 'server/server' path ? it would be more consistent across projects

**Comment by cRicateau** [comment]
@BenEllerby Could you add a documentation on how to use it ? coud you create a installation-django.md file ?

File: generators/app/templates/django_bootstrap.sh (hunk 1 above)
Comment (BenEllerby) [reply generators/app/templates/django_bootstrap.sh]: I agree!

File: generators/app/index.js (hunk 2 above)
Comment (BenEllerby) [reply generators/app/index.js]: Yes

File: generators/app/index.js (hunk 3 above)
Comment (BenEllerby) [reply generators/app/index.js]: In django inside your project you have app folders.
One of those app folders needs to be your 'main' app, therefore I normally create an app with the name of the project to be the core application with the settings for the project.

**Comment by ThierryCols** [comment]
I'm on it boys

**Pull request closed without merging**